core/services/configuration_generator.rb

```diff
@@ -30,12 +30,10 @@ def self.generate_json_format(box_definitions, name, product_configs, recipes_na
   end
 
   # Check whether box needs to be subscribed or not
-  # @param box_definitions [BoxDefinitions] the list of BoxDefinitions that are configured in the application
-  # @param box [String] name of the box
-  def self.check_subscription_manager(box_definitions, box)
```

Comment:
This change must be reversed. If you have changed the way data comes to the outer method, it does not need to seep to the internal methods.